bigquery/indexer.py

```diff
@@ -439,6 +406,44 @@ def create_samples_json_export_file(es, storage_client, index_name,
     logger.info('Wrote gs://%s/%s' % (bucket_name, samples_file_name))
 
 
+def fix_samples_data_for_es(participant_docs):
+    """
+    Currently our samples are stored as a dictionary keyed by sample_ids:
+    samples: {
+        sample_id_1: {
+            SAMPLE_ID: sample_id_1
+            column_name_1: value_a,
+            column_name_2: value_b,
+        },
+        sample_id_2: {
+            SAMPLE_ID: sample_id_2,
+            column_name_1: value_c,
+            column_name_2: value_d,
+        }
+    }
+    We need to change it to the structure readable by elasticsearch
+    samples: [
+        {
+            SAMPLE_ID: sample_id_1
+            column_name_1: value_a,
+            column_name_2: value_b,
+        },
+        {
+            SAMPLE_ID: sample_id_1
+            column_name_1: value_a,
+            column_name_2: value_b,
+        }
+    ]
+    """
```

Comment:
Add comment:

We use the first format for indexing so that if there are two sample tables containing information on the same sample, we can look up the sample when indexing the second table.